### PR TITLE
RD-754 Added labels widget description

### PR DESCRIPTION
**Pull request by kubama**

### CODE REVIEW

File: content/working_with/console/widgets/labels.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+* **Key** - label key
+* **Value** - label value
+* Actions column, containing value edit and label delete action icons
+
+![labels]( /images/ui/widgets/labels.png )
+
+By clicking buttons above the table you can execute the following operations:
+
+* `Add` - opens labels add modal
```
Comment (qooban) [inline content/working_with/console/widgets/labels.md:18]: That's only visible if user has manage labels permission, right?

**Review by qooban — COMMENTED** [review]
Clean and straightforward 👍🏻 

Added some comments, mostly related to small additions and alignments with other widgets' descriptions.

**Comment by kubama** [comment]
@qooban All comments addressed

File: content/working_with/console/widgets/deploymentActionButtons.md [hunk 2]
```diff
@@ -4,10 +4,9 @@ title: Deployment action buttons
 category: Widgets
 draft: false
 ---
-Buttons which allow running workflows of a specific deployment, updating it or deleting it. The deployment can be selected in one of the following ways:
+Buttons which allow running workflows of the deployment currently in the context, updating it or deleting it.
```
Comment (qooban) [inline content/working_with/console/widgets/deploymentActionButtons.md:7]: I suspect you wanted to say:
```suggestion
Buttons which allow running workflows of the deployment currently set in the context, updating it or deleting it.
```
Am I right?

**Review by qooban — COMMENTED** [review]
Big thanks for extracting context part 👍🏻 

Just one comment.

File: content/working_with/console/widgets/deploymentActionButtons.md (hunk 2 above)
Comment (kubama) [reply content/working_with/console/widgets/deploymentActionButtons.md:7]: My impression is that 'set' is not required there, but of course I have nothing against it - applying.

**Review by qooban — APPROVED** [review]

File: content/working_with/console/widgets/labels.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+layout: bt_wiki
+title: Labels
+category: Widgets
+draft: false
+---
+
+Displays list of labels defined for deployment currently in the context (see [notes]({{< relref "working_with/console/widgets/_index.md" >}}) for more information on resource context).
```
Comment (ofercloudify) [inline content/working_with/console/widgets/labels.md:8]:
```suggestion
Displays the list of labels defined for the deployment which is currently in the context (see [notes]({{< relref "working_with/console/widgets/_index.md" >}}) for more information on resource context).
```

File: content/working_with/console/widgets/labels.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* **Key** - label key
+* **Value** - label value
+* Actions column, containing value edit and label delete action icons, available only to users with `deployment_create` permission
+
+![labels]( /images/ui/widgets/labels.png )
+
+By clicking buttons above the table you can execute the following operations:
+
+* `Add` - opens labels add modal, available only to users with `deployment_create` permission
```
Comment (ofercloudify) [inline content/working_with/console/widgets/labels.md:19]:
```suggestion
* `Add` - opens the labels add modal, available only to users with `deployment_create` permission
```

File: content/working_with/console/widgets/labels.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+* `Export` - exports table content into JSON file
```
Comment (ofercloudify) [inline content/working_with/console/widgets/labels.md:20]:
```suggestion
* `Export` - export the table content into a JSON file
```

File: content/working_with/console/widgets/labels.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+You can find more about labels [here]({{< relref "cli/orch_cli/deployments.md#labels" >}}).
```
Comment (ofercloudify) [inline content/working_with/console/widgets/labels.md:22]:
```suggestion
You can learn more about labels [here]({{< relref "cli/orch_cli/deployments.md#labels" >}}).
```

File: content/working_with/console/widgets/topology.md [hunk 7]
```diff
@@ -5,12 +5,9 @@ category: Widgets
 draft: false
 ---
 
-Displays the topology of a selected blueprint or deployment.
+Displays the topology of blueprint or deployment currently set in the context.
```
Comment (ofercloudify) [inline content/working_with/console/widgets/topology.md:8]:
```suggestion
Displays the topology of the blueprint or the deployment currently set in the context.
```

**Review by ofercloudify — CHANGES_REQUESTED** [review]

**Review by qooban — APPROVED** [review]

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**